proto/api_v3/query_service.proto

```diff
@@ -40,47 +40,64 @@ message GetTraceRequest {
   bool raw_traces = 4;
 }
 
-// Query parameters to find traces. Except for num_traces, all fields should be treated
-// as forming a conjunction, e.g., "service_name='X' AND operation_name='Y' AND ...".
+// Query parameters to find traces. All fields should be treated
```

Comment:
There are normative sentences, either combine them into a single paragraph, or a separate paragraph from the title (such format should be consistent throughout), or a bullet list.

Reply:
@yurishkuro let me know if thats better